### PR TITLE
Requesting signoff for Main Control

### PR DESCRIPTION
**Pull request by gabeakim**
Requesting signoff for Main Control

### CODE REVIEW

**Review by DrVanNeste — COMMENTED** [review]
The figures look pretty good now, but you need to show the simulations and show what the current is. There should be a simulated graph - we talked at length about this on Friday, I don't understand why your systems keep coming up short on information after we discuss at length about it.

**Review by gabeakim — COMMENTED** [review]
I am requesting Approval 2/1/2023

File: Documentation/Signoffs/Main_Control.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Main Control Subsystem
+
```
Comment (DrVanNeste) [inline Documentation/Signoffs/Main_Control.md:2]: Hi Gabe,

There are arrows leading into this figure that need to have labels so we know what they are.

**Review by DrVanNeste — APPROVED** [review]
I approve.

**Pull request merged**